addon/index.js

```diff
@@ -152,10 +152,13 @@ export class EmberChangeset extends BufferedChangeset {
   /**
    * Executes the changeset if in a valid state.
    *
+   * @param {{ force: boolean }}
    * @method execute
    */
-  execute() {
-    if (this.isValid && this.isDirty) {
+  execute({ force = false} = {}) {
```

Comment:
`r/{ force = false}/{ force = false }`

but b/c no lint rule, not necessary if you don't feel like it!

Reply:
My bad, will fix :)